packages/rich-text/src/plugins/Hyperlink/index.tsx

```diff
@@ -285,5 +287,28 @@ export const createHyperlinkPlugin = (sdk: FieldExtensionSDK): RichTextPlugin =>
         },
       },
     ],
+    normalizer: [
+      {
+        match: {
+          type: INLINES.HYPERLINK,
+        },
+        validNode: hasText,
+        transform: transformRemove,
```

Comment:
* You can drop the `transformRemove` part because it's the default transformer 😉
* Another option would be to have a single `normalizer` rule similar on the top level like this:

```javascript
normalizer: [
  {
    match: {
      type: [INLINES.HYPERLINK, INLINES.ENTRY_HYPERLINK, INLINES.ASSET_HYPERLINK]
    },
    validNode: hasText,
  },
],
```

Reply:
- I think it's good to be explicit so it matches my expectations when looking at the code to know what's going on.
- I was wondering if adding an array with multiple types would work, thanks

Reply:
> I think it's good to be explicit so it matches my expectations when looking at the code to know what's going on.

Fair enough. In my head, I think of it like: "Every invalid element should be removed, tell me when there is an exception"